extensions/amp-inline-gallery/1.0/pagination.js

```diff
@@ -15,42 +15,61 @@
  */
 
 import * as Preact from '../../../src/preact';
-import * as styles from './pagination.css';
 import {CarouselContext} from '../../amp-base-carousel/1.0/carousel-context';
 import {useContext} from '../../../src/preact';
+import {useStyles} from './pagination.jss';
 
 /**
  * @param {!InlineGalleryDef.PaginationProps} props
  * @return {PreactDef.Renderable}
  */
 export function Pagination({inset, ...rest}) {
+  const classes = useStyles();
   const {slideCount, currentSlide, setCurrentSlide} = useContext(
     CarouselContext
   );
   const Comp = slideCount <= 8 ? Dots : Numbers;
   const indicator = (
     <div
       aria-hidden="true"
-      class="i-amphtml-carousel-pagination-container"
+      class={`${classes.container} ${inset ? classes.inset : ''}`}
       style={{
-        ...styles.paginationContainer,
-        ...(inset ? styles.insetPaginationContainer : {}),
+        // TODO: where does this belong?
```

Comment:
What is 'this'?

Reply:
This is actually more of a question to you :) But it seemed like a constant `height` is not desirable here? I kept it "as is" for now.

Reply:
I was asking what the TODO refers to, thanks for clarifying on the constant height. It should be left as is since the dots themselves should be 20px tall; if the user wants to specify a separate height it will be set on the containing div, not the indicator UI itself.

Reply:
Ok. So sounds like I can drop `TODO` then and the `height` belongs in the `jss` file. Right?

Reply:
SGTM

Reply:
Fixed. I created a `Wrapper` component - a non-contained counterpart to the `ContainWrapper` to make the style propagation more canonical.